lib/blocs/home_page_bloc.dart

```diff
@@ -0,0 +1,74 @@
+import 'dart:convert';
+
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:trufi_core/models/map_route_state.dart';
+import 'package:trufi_core/repository/local_repository.dart';
+import 'package:trufi_core/repository/request_manager.dart';
+import 'package:trufi_core/trufi_models.dart';
+
+class HomePageBloc extends Cubit<MapRouteState> {
```

Comment:
A suggestion, We could rename from HomePageBloc to HomePageCubit, naming it Cubit or Bloc is not relevant? 👍

Reply:
Good catch :)